src/Form/Extension/AceEditor/Type/AceEditorType.php

```diff
@@ -34,14 +38,12 @@ public function __construct(bool $useStimulus)
 
     public function configureOptions(OptionsResolver $resolver): void
     {
-        // Remove id from ace editor wrapper attributes. Id must be generated.
+        // Remove id from ace editor wrapper attributes, it must be generated.
         $wrapperAttrNormalizer = static function (Options $options, mixed $aceAttr): array {
-            if (\is_array($aceAttr)) {
-                if (\array_key_exists('id', $aceAttr)) {
-                    unset($aceAttr['id']);
-                }
-            } else {
+            if (!\is_array($aceAttr)) {
                 $aceAttr = [];
+            } else {
+                unset($aceAttr['id']);
```

Comment:
unset does not generate warnings on undefined array indexes, making the array_key_exists check unnecessary:

```
php -d error_reporting=-1 -r '$a = []; unset($a["f"]);'
```